fastchat/serve/gradio_web_server.py

```diff
@@ -148,11 +148,12 @@ def get_model_list(controller_url, register_api_endpoint_file, multimodal):
     if register_api_endpoint_file:
         api_endpoint_info = json.load(open(register_api_endpoint_file))
```

Comment:
Set default value for the entries of endpoint registry. Specifically, the default value of "text-arena" should be True so that it is compatible with the old endpoints file.